torch/_inductor/__init__.py

```diff
@@ -143,7 +146,8 @@ def aot_compile(
         options:  Optional dict of config options.  See `torch._inductor.config`.
 
     Returns:
-        Path to the generated shared library
+        Path to the generated shared library, or a list of files generated by
```

Comment:
This feels like a complicated API, why not always return a list, albeit for .so it will be a single element list

Reply:
I don't want to break any existing callsites 😅 but yes! I can fix the rest of the callsites to return a list in a followup.